gstreamer-sdp/src/s_d_p_zone.rs

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2018 Sebastian Dröge <sebastian@centricular.com>
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use auto::SDPResult;
+
+use ffi;
+
+pub struct SDPZone(ffi::GstSDPZone);
+
+impl SDPZone {
+    pub fn set(&mut self, adj_time: &str, typed_time: &str) -> SDPResult {
```

Comment:
This is part of the reason why this (and others can't be autogenerated). set/clear are actually constructor and destructor here, and the `SDPResult` return value should become some kind of `Result<_,_>` (I don't think `clear` can even fail). So there should be a `fn new(adj_time, typed_time) -> Result<Self, _>`, and `clear` just is called in `Drop::drop`.

Also there should be accessors for the fields of these structs